luigi/contrib/bigquery.py

```diff
@@ -33,6 +33,8 @@
     logger.warning('Bigquery module imported, but google-api-python-client is '
                    'not installed. Any bigquery task will fail')
 
+IMPORT_TIME_SALT = random.randrange(0, 1e10)
```

Comment:
Ah, I see! Can you add that explenation @DeaconDesperado? Then I think this can be merged :)